src/core/Tooltip/index.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  makeStyles,
+  Tooltip as RawTooltip,
+  TooltipProps,
+} from "@material-ui/core";
+import React from "react";
+import { AppThemeOptions } from "../styles";
+
+export { TooltipProps };
+
+const useStyles = makeStyles((theme: AppThemeOptions) => {
+  return {
+    arrow: {
+      "&::before": {
+        border: `1px solid ${theme?.app?.colors.gray["300"]}`,
+      },
+      color: "white",
+    },
+    tooltip: {
+      backgroundColor: "white",
+      border: `1px solid ${theme?.app?.colors.gray["300"]}`,
+      boxShadow: theme?.app?.shadows["m"],
+      color: "black",
+      padding: theme?.app?.spacing.l,
+      fontSize: "13px",
+      lineHeight: "20px",
+      letterSpacing: "0.3px",
+      fontWeight: theme?.app?.fontWeights.regular,
```

Comment:
This corresponds to `font-body-xs`, but I'm not sure how to apply the mixin while using `makeStyles`.

I also tried applying it with emotion like this: 
```
export const StyledTooltip = styled(Tooltip)`
  &.MuiTooltip-tooltip {
    ${fontBody("xs")};
  }
`;
```
but couldn't get the style to apply. I'm pretty sure the class name is `.MuiTooltip-tooltip` based on the [docs](https://material-ui.com/api/tooltip/#css), so not sure why it's not showing up.

Any suggestions on how to clean this up are greatly appreciated! @tihuan

Reply:
Hurray! Thanks so much for the awesome PR, @j-x-han 🎉 🙏 !

I have been doing some research on this new use case and might have a solution 🤞 Will send out a PR against this one in a bit 😄 Thank you!